<launch>

	<arg name="input_scan"/>
	<arg name="output_scan"/>

	<node pkg="laser_filters" type="scan_to_scan_filter_chain" name="laser_filter" ns="/bender/sensors/" output="screen">
		<rosparam command="load" file="$(find uchile_laser_pipeline)/config/pipeline.yaml" />

		<!-- incoming scan -->
		<remap from="scan" to="$(arg input_scan)" />

		<!-- output scan -->
		<remap from="scan_filtered" to="$(arg output_scan)" />

	</node>
</launch>